data/json/mapgen/dairy_farm.json

```diff
@@ -0,0 +1,251 @@
+[
+  {
+    "type":"monstergroup",
+    "name" : "GROUP_COWS",
+    "is_safe" : true,
+    "default" : "mon_null",
+    "monsters":[{"monster" : "mon_cow", "cost_multiplier" : 10, "freq": 100, "ends" : 168 },
+    {"monster" : "mon_cow", "freq": 50, "cost_multiplier" : 5, "starts" :  336  }
+    ]
```

Comment:
Fixed the time gap.

Didn't know spawn cost was that costly. Also what do you mean by 10? and 5%? should they spawn 100% and 50% of the time? I thought it was tied to freq.